src/utils/fs.spec.js

```diff
@@ -1,39 +1,189 @@
+const fs = require('fs')
+const path = require('path')
+
+jest.mock('fs')
+jest.mock('path')
+jest.mock('util', () => ({
+  promisify: jest.fn().mockImplementation(fn => fn),
+}))
+
+const toPathObject = (directory, filename) => ({
```

Comment:
This method only seems to be called once - maybe it would be clearer if the object is constructed at that place in the test suite? WDYT?

Reply:
No need for that function at all, since path will not be mocked based on your feedback.